src/rt/sections_osx.d

```diff
@@ -23,6 +23,13 @@ import core.sys.osx.mach.getsect;
 import rt.deh, rt.minfo;
 import rt.util.container.array;
 
+version (D_LP64)
+    import rt.sections_osx64;
+else
+    import rt.sections_osx32;
+
+mixin SectionsOSX;
```

Comment:
The reason I did it like this is because the code in `sections_osxXX` depends on code in `sections_osx`. I tried to avoid a circular dependency without changing too much of the code. Not sure if this is better though.